content/compose/compose-file/05-services.md

```diff
@@ -181,6 +182,9 @@ cap_drop:
 ```
 
 ## cgroup
+
+{{< introduced compose 2.15.0 "../release-notes.md#2150" >}}
```

Comment:
Not sure why but in the [deploy preview](https://deploy-preview-19773--docsdocker.netlify.app/compose/compose-file/05-services/#cgroup) I don't see this under the `cgroup` heading.